geometric_kernels/spaces/base.py

```diff
@@ -87,6 +87,7 @@ def __call__(self, X: B.Numeric, **parameters) -> B.Numeric:
         Phi = take_along_axis(self.eigenvectors, indices, axis=0)
         return Phi
 
+    @property
```

Comment:
This will break `num_eigenfunctions()` calls, right? Didn't we have those? Is this done to unify the overall style?